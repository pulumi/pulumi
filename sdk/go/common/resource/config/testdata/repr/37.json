{"my:foo":{"bar":"baz","secure":"myvalue"}}