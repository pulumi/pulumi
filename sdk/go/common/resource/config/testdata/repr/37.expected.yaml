decrypt:
  my:foo: '{"bar":"baz","secure":"myvalue"}'
paths:
  my:foo:
    value:
      bar: baz
      secure: myvalue
    string: '{"bar":"baz","secure":"myvalue"}'
    redacted: '{"bar":"baz","secure":"myvalue"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAw/4ItAAIGc2VjdXJlBnN0cmluZwwJAAdteXZhbHVlA2JhcgZzdHJpbmcMBQADYmF6
    secure: false
    isObject: true
  my:foo.bar:
    value: baz
    string: baz
    redacted: baz
    object: EBAABnN0cmluZwwFAANiYXo=
    secure: false
    isObject: false
  my:foo.secure:
    value: myvalue
    string: myvalue
    redacted: myvalue
    object: FBAABnN0cmluZwwJAAdteXZhbHVl
    secure: false
    isObject: false
